packages/server/src/fhir/operations/utils/agentutils.ts

```diff
@@ -121,9 +123,13 @@ export async function handleBulkAgentOperation(
           agents[i]
         )
       );
+      failureCount += 1;
       continue;
     }
     const [outcome, params] = result.value;
+    if (!isOk(outcome)) {
+      failureCount += 1;
```

Comment:
I recommend we save the `outcome` here and use it to determine status.

`++`?

Reply:
It's possible that you get multiple different types of outcomes, should we just save the first one?